spectree/plugins/flask_plugin.py

```diff
@@ -187,11 +188,28 @@ def validate(
             after(request, response, req_validation_error, None)
             abort(response)
 
-        response = make_response(func(*args, **kwargs))
+        result = func(*args, **kwargs)
+
+        status = 200
+        rest = []
```

Comment:
I think we also need to define "model = result" here?

Reply:
The `if/if/else/else` below that will set `model`
But only the `if/if` will set `status` and `rest` so these are just the defaults for the other branches.

Reply:
How about the `model` in line:203?

Let's say the user returns a "pydantic.BaseModel" instance, it won't go to the `if/if/else/else`. But we can still skip it if we check in the next `if`.

Reply:
I see. You're right. I missed the "else" part.